drltrace/drltrace_frontend.cpp

```diff
@@ -302,11 +308,25 @@ _tmain(int argc, const TCHAR *targv[])
         tmp--;
     *(tmp+1) = '\0';
 
+    /* in case of default config option, we use drltrace's install path */
+    if (op_config_file.get_value().compare(DEFAULT_CONFIG_DIR) == 0) {
```

Comment:
Rather than this odd sentinel value, why not just have the default be relative to the binary as Dr. Memory does for its config files (see obtain_configfile_path())?  That seems cleaner.